src/game/Spell.cpp

```diff
@@ -3371,6 +3371,13 @@ void Spell::SendLogExecute()
         {
             switch (m_spellInfo->Effect[EFFECT_INDEX_0])
             {
+                case SPELL_EFFECT_DISPEL:
+                case SPELL_EFFECT_THREAT:
```

Comment:
I see it uses
`data << unit->GetPackGUID();`
where you use
`data << unit->GetObjectGuid();`

Hmmm, my comment may be bullocks then, but I still think it's worth the try if you havn't already.